src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/RememberMeFactory.php

```diff
@@ -94,31 +99,66 @@ public function create(ContainerBuilder $container, string $id, array $config, ?
 
     public function createAuthenticator(ContainerBuilder $container, string $firewallName, array $config, string $userProviderId): string
     {
-        $templateId = $this->generateRememberMeServicesTemplateId($config, $firewallName);
-        $rememberMeServicesId = $templateId.'.'.$firewallName;
+        if (!$container->hasDefinition('security.authenticator.remember_me')) {
```

Comment:
Previously, security loaded all services in the main DI config of the SecurityBundle. We're now only loading services whenever a factory is used (to avoid "useless" services in the dev environment)